tasmota/support_tasmota.ino

```diff
@@ -351,7 +358,25 @@ void SetLedPowerIdx(uint32_t led, uint32_t state)
     } else {
       led_power &= (0xFF ^ mask);
     }
-    DigitalWrite(GPIO_LED1, led, bitRead(led_inverted, led) ? !state : state);
+	uint16_t led_pwm_set = 0;
+	if (bitRead(Settings.ledpwm_mask, led)) {
+		if (bitRead(led_inverted, led)) {
+		  if (state) {
+			led_pwm_set = Settings.pwm_range - Settings.ledpwm_on;
+		  } else {
+			led_pwm_set = Settings.pwm_range - Settings.ledpwm_off;
+		  }
+		} else {
+		  if (state) {
+			led_pwm_set = Settings.ledpwm_on;
+		  } else {
+			led_pwm_set = Settings.ledpwm_off;
+		  }
+		}
+		analogWrite(Pin(GPIO_LED1, led), led_pwm_set);
```

Comment:
You can mimic the full version of `xdrv_04_light.ino`:

```
cur_col = cur_col > 0 ? changeUIntScale(cur_col, 0, Settings.pwm_range, Light.pwm_min, Light.pwm_max) : 0;   // shrink to the range of pwm_min..pwm_max
analogWrite(Pin(GPIO_PWM1, i), bitRead(pwm_inverted, i) ? Settings.pwm_range - cur_col : cur_col);
```

with full support of range and pwm_min/pwm_max